inc/feedcreator.class.php

```diff
@@ -505,7 +510,7 @@ function addItem($item) {
      * @param int     $length the maximum length the string should be truncated to
      * @return string    the truncated string
      */
-    function iTrunc($string, $length) {
+    static function iTrunc($string, $length) {
```

Comment:
It is an annotation in the PHPDocs, it is also suggested by IntelliJ IDEA.
Further this function is indeed everywhere called as static function in DokuWiki.